src/components/SponsorList.tsx

```diff
@@ -1,19 +1,22 @@
 import React from "react";
 import * as SponsorLogos from "../images/sponsors";
 
-const sponsors = [{ name: "Vaco", logo: SponsorLogos.Vaco }];
+const sponsors = [
+  { name: "Vaco", logo: SponsorLogos.Vaco },
+  { name: "Pluralsight", logo: SponsorLogos.Pluralsight },
+];
 
 const SponsorList: React.FC<{}> = () => {
   return (
-    <div className="w-screen bg-slate-200 p-8 text-center">
+    <div className="w-screen bg-slate-200 py-8 px-2 text-center">
       <h3>Our Sponsors</h3>
-      <div className="flex flex-row flex-wrap justify-center p-8">
+      <div className="flex flex-row flex-wrap justify-center py-8 px-2">
         {sponsors.map((sponsor) => (
           <img
             key={sponsor.name}
             alt={`${sponsor.name} logo`}
             src={sponsor.logo}
-            className="w-60 md:w-30 p-4 m-2 bg-white"
+            className="h-16 md:h-24 p-4 m-2 bg-white rounded-lg"
```

Comment:
Changed some of these around because the Pluralsight logo is WIDE compared to Vaco. Should work better to restrict the height instead of the width.